.github/workflows/actions.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Lint and Test
+    steps:
+    - uses: actions/checkout@v2
+      name: Check out repository
+    - uses: actions/setup-node@v2
+      name: Setup Node.js
+      with:
+        node-version: '16.6.0'
```

Comment:
why not the LTS? 
![image](https://user-images.githubusercontent.com/22256912/129730628-137ae0b7-b291-4839-b06e-7701030413a8.png)